demo/src/screens/CheckoutScreen/CheckoutScreen.js

```diff
@@ -22,9 +25,10 @@ const BILLING_FORM_ID = 'billing-info-form';
 /**
  * Page component for a user to enter in personal billing information
  * and confirm the items in their cart.
+ * @param {function()} clearCart A function to clear all items from cart.
  * @return {!JSX}
  */
-export function CheckoutScreen() {
+const CheckoutScreenBase = function({clearCart}) {
```

Comment:
Nit: Can you make this an arrow function? This is usually preferred over the function keyword in the style guide:
https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions

Reply:
Done! I also changed this pattern elsewhere and did some manual testing to ensure it didn't break anything. Thanks for the suggestion.

Reply:
Nice, thanks!